x-pack/plugins/apm/server/lib/apm_telemetry/collect_data_telemetry/tasks.ts

```diff
@@ -1159,13 +1159,16 @@ export const tasks: TelemetryTask[] = [
     name: 'per_service',
     executor: async ({ indices, search }) => {
       const response = await search({
-        index: [indices.metric],
+        index: [indices.transaction],
```

Comment:
Transaction events are subject to sampling the returned stats are not going to be accurate. If we are ok with that I don't have any objections.

Reply:
In this case we're not paying attention to any of the stats. We pretty much just query metadata from the transaction document, no counts or anything.